src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileTarget.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
revert this.

Reply:
As part of next push, I'll squash and merge which will allow me to revert it.